### PR TITLE
Update purchase subscription docs to include advanced section

### PR DESCRIPTION
**Pull request by jneterer**
This PR adds documentation for advanced settings for the `purchaseSubcription` request. You can find the `userbase-js` PR with more information [here](https://github.com/smallbets/userbase/pull/338#issue-832395952).

### CODE REVIEW

File: src/pages/docs_sdk_purchase-subscription.html [hunk 1]
```diff
@@ -67,6 +67,24 @@ <h3 id="notes">Notes</h3>
...
+  <ul>
+    <li>You can support calculating tax at checkout. To enable automatic tax:
+      <ul>
+        <li>Add a coupon to your Stripe account under <a href="https://dashboard.stripe.com/coupons" target="_blank">Products > Coupons</a></li>
+        <li>Then add a promotion code for that coupon</li>
+      </ul>
+    </li>
+    <li>You can allow promo codes at checkout. To enable promo codes:
+      <ul>
+        <li>You need to activate tax collection</li>
```
Comment (j-berman) [inline src/pages/docs_sdk_purchase-subscription.html:81]: Small nit: this is done via the same link as in the next bullet, think it makes sense to include the link in this first bullet instead

File: src/pages/docs_sdk_purchase-subscription.html [hunk 2]
```diff
@@ -67,6 +67,24 @@ <h3 id="notes">Notes</h3>
     </li>
   </ul>
 
+  <h3 id="advanced">Advanced</h3>
+
+  <ul>
+    <li>You can support calculating tax at checkout. To enable automatic tax:
```
Comment (j-berman) [inline src/pages/docs_sdk_purchase-subscription.html:73]: Seems you mixed up this line with the other :)

**Review by j-berman — COMMENTED** [review]
Small stuff

**Comment by jneterer** [comment]
@j-berman Great catch on both of those! I've made the requested updates.

**Comment by j-berman** [comment]
Sorry last thing that just hit me: we should also add a bullet to the end of each saying that the final step is to enable the respective feature via the "advanced" section in the app's Payments Portal in the admin panel (and link to it), or highlight how this is the final step in some capacity

**Comment by jneterer** [comment]
@j-berman I'll go ahead and make this update, however I'm not sure it'll be possible to link to it, only because the link would require the userbase app id, which of course we don't know in the docs.

File: src/pages/docs_sdk_purchase-subscription.html [hunk 3]
```diff
@@ -67,6 +67,24 @@ <h3 id="notes">Notes</h3>
...
+        <li>Add a coupon to your Stripe account under <a href="https://dashboard.stripe.com/coupons" target="_blank">Products > Coupons</a></li>
+        <li>Then you need to set tax behavior on a product</li>
+      </ul>
+    </li>
+    <li>You can allow promo codes at checkout. To enable promo codes:
+      <ul>
+        <li>You need to activate tax collection under <a href="https://dashboard.stripe.com/settings/tax/activate" target="_blank">Settings > Tax Settings</a></li>
+        <li>Set an origin address via this link and configure tax settings</li>
+        <li>Then add a promotion code for that coupon</li>
+      </ul>
```
Comment (j-berman) [inline src/pages/docs_sdk_purchase-subscription.html:84]: Something like:

```
<li>Finally, visit your app's Payments Portal in your <a href="https://v1.userbase.com/" target="_blank">Admin panel</a>, click "Show Advanced", and enable the option to "Allow promotion codes in Checkout"</li>
```

for both this and tax. Whatever you think feels right :)

Comment (jneterer) [reply src/pages/docs_sdk_purchase-subscription.html:86]: @j-berman Done!

**Pull request merged**